FlowCrypt/Core/Core.swift

```diff
@@ -223,7 +217,7 @@ final class Core: KeyDecrypter, KeyParser, CoreComposeMessageType {
     }
 
     func handleCallbackResult(json: String, data: [UInt8]) {
-        cb_last_value = (json, data)
+        callbackResults.append((json, data))
     }
```

Comment:
How is it different? If this class is an actor, then single value and array is equivalent, since a single call can never produce more than one callback.

Reply:
You can run several `Core` methods in parallel and one of them will override value of another since you have only single value